src/doc/templates/templates_term.nit

```diff
@@ -0,0 +1,324 @@
+# This file is part of NIT ( http://www.nitlanguage.org ).
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Markdown templates for Nit model MEntities.
+#
+# This module introduces Markdown rendering services in model entities.
+# With the `no_color` option set to `false`, the output can be highlighted for console.
+module templates_term
+
+import model_collect
+import console
+
+redef class MDoc
+	# Returns the full comment formatted for the console
+	fun cs_comment(no_color: nullable Bool, indent: nullable Int): String do
```

Comment:
a lot of your function have a `no_color` parameter. I'm worrying that someday you may want to add more options/flag.
What about having a single config parameter where you can add more flags/option, and even delegate some services?

Reply:
Let's refactor it when needed.